sets update for tabbed examples with bikes (#2496)
redis/redis-doc: docs/data-types/sets.md

(integer) 2
> SADD bikes:racing:usa bike:1 bike:4
(integer) 2
{{< /clients-example >}}

* Check whether bike:1 or bike:2 are racing in the US.
{{< clients-example sets_tutorial sismember >}}
> SISMEMBER bikes:racing:usa bike:1
(integer) 1
> SISMEMBER bikes:racing:usa bike:2
(integer) 0
{{< /clients-example >}}

* Which bikes are competing in both races?
{{< clients-example sets_tutorial sinter >}}
> SINTER bikes:racing:france bikes:racing:usa
1) "bike:1"
{{< /clients-example >}}

* How many bikes are racing in France?
{{< clients-example sets_tutorial scard >}}
> SCARD bikes:racing:france
(integer) 3
{{< /clients-example >}}
## Tutorial

The `SADD` command adds new elements to a set. It's also possible
to do a number of other operations against sets like testing if a given element
already exists, performing the intersection, union or difference between
multiple sets, and so forth.

{{< clients-example sets_tutorial sadd_smembers >}}
> SADD bikes:racing:france bike:1 bike:2 bike:3
(integer) 3
> SMEMBERS bikes:racing:france
1) bike:3
2) bike:1
3) bike:2
{{< /clients-example >}}

Here I've added three elements to my set and told Redis to return all the

> elements. There is no order guarantee with a set -- Redis is free to return the
> elements in any order at every call, since there is no contract with the
> user about element ordering.

Reviewer: ```suggestion
elements. There is no order guarantee with a set. Redis is free to return the
elements in any order at every call.
```

> Redis has commands to test for membership. This can be both a single item
> or a variadic command to test membership for multiple items at once:

Reviewer: ```suggestion
Redis has commands to test for set membership. These commands can be used on single as well as multiple items:
```
